test/runTests.cpp

```diff
@@ -10,16 +10,23 @@
 using namespace std;
 using namespace edlib;
 
+template <class Element, class AlphabetIdx, AlphabetIdx AlphabetSize>
```

Comment:
I am guessing alphabetSize could have also just been an argument of the function, right? If that is correct, how did you decide to put it as template parameter instead? I am not saying it is wrong, I am just wondering. I guess this way you get it to during compile time instead of runtime, but I don't think it really makes any difference.

Reply:
I wanted to put it as an argument of the function but I remembered you said in the last comment on the previous PR that we can add it as a template parameter. I didn't find any difference so I did what you suggested but I think it is reasonable to have it as an argument. I'll change it.

Reply:
Yes I said that, true! But I wasn't really thinking too deeply about it I guess :D. Both is fine really, but ok, I agree argument maybe makes a little bit more sense.